fields.yml

```diff
@@ -396,35 +396,64 @@
             Unique ID to describe the event.
           example: 8a4f500d
     
+        - name: kind
+          level: core
```

Comment:
Sorry, only spotted now but these fields should go into extended for now.

Reply:
I moved `event.kind` and `event.outcome` to extended.

Reply:
I'll be proposing pushing these to core sometime in the future :-)